src/server/game/Entities/Player/Player.h

```diff
@@ -2148,6 +2148,8 @@ class TC_GAME_API Player : public Unit, public GridObject<Player>
 
         std::string GetDebugInfo() const override;
 
+        uint32 GetFreeInventorySpace() const;
```

Comment:
Instead of at the end, try to place it near similar/related methods please

Reply:
Done, should be fine now.